### ISSUE TITLE
Document recommendation to have GPU on scheduler if workers have GPU

### ISSUE THREAD

**Issue by ntabris** [dask/distributed#6924]
In the discussion on https://github.com/dask/distributed/pull/6908 it emerged that it's recommended to have a GPU on the scheduler if you're using GPU workers.

As far as I can tell, that's not documented anywhere. It would be super if there was also some explanation of why this is recommended.

I don't know enough to write that so tagging @jacobtomlinson @pentschev @quasiben because they're the people who said this was best practice.

**Comment by jacobtomlinson** [comment]
Thanks @ntabris. We are just discussing this at the RAPIDS Dask team sync and will come up with a plan.

cc @dask/gpu

**Comment by jakirkham** [comment]
To briefly summarize the issue, there can be GPU objects in the Dask graphs (particularly HLGs). If these are deserialized on the Scheduler (as happens with HLGs), they will require a GPU for the memory allocation and object construction.

It sounds like our current leaning is the same as this issue (documenting this somewhere). We are still trying to figure out the right home for this documentation.

Anyone should feel free to add to/correct the summary above :)

**Comment by jacobtomlinson** [comment]
@quasiben also mentioned that if you call `dask.submit` with objects that are in GPU memory they will also be deserialized on the scheduler and need some GPU memory to be unpacked into.

**Comment by jakirkham** [comment]
Yeah it is not exclusively an HLG problem. If a user created a GPU object on the Client side and sent this over, the Scheduler may touch this in some cases, which could present the same kind of issue.

**Comment by jrbourbeau** [comment]
> We are still trying to figure out the right home for this documentation.

https://docs.dask.org/en/stable/gpu.html seems like a clear candidate for where to put this information (other suggestions certainly welcome though)
